### PR TITLE
Complete Hello Microverse Project

### PR DESCRIPTION
**Pull request by David-Kasilia**
- Reviewed the changes required.
- Renamed all the branches to more descriptive names.
- Added the project summary below.
- Added Title to HTML.
- Created the heading using the h1 element, and assigned a class to the element.
- Used external CSS to link to the HTML.
- Used the class selector to style the h1 element.
- Added descriptive information about the project in the README file.
- Added the .gitignore file to ignore unnecessary files.
- Ensured that the test.MD had the text: This file should be ignored by git and that it is not visible in the Hello Microverse Project directory.
- Fixed the stylelint errors.
- Updated the Readme File

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+Clone the directory to local repository and make it available locally.<br>
+From the main branch,create a branch eg:  file-creation branch  to create the necessary files required for the project which will be merged to the main branch. These Files include index.html and style.css.<br>
+Then create-linter-files branch  to create the linter files. Then later will be merged to the main branch.<br>
+Open VSCode, choose open folder and open the directory that was cloned from Github.<br>
+Open the index.html and populate it with boiler plate code and a h1 element with the text Hello Microverse and assign a class to it of name header
+Link the stylesheet to index.html.<br>
+Open the style.css and add the style properties to the h1 element.
+
+# Run Tests
+Run the index.html on your preferred browser or use a live server
```
Comment (HAMMAS-SALEEM) [inline README.md:28]: - [x] You have added a readme file great :+1: but I think you forgot some headings and their content. Please compare your readme with this [repo readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse) and kindly add the missing headings and their content :thinking:

File: stylesheet/style.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.header {
+    color: purple;
+    text-align: center;
+    font-size: 62px;
+    font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (HAMMAS-SALEEM) [inline stylesheet/style.css:6]: third hello microverse linters

-  [x] It's great that you have implemented the linters correctly :+1: But your linters are not passing all the tests. Please check your linter errors and kindly try to fix them :innocent:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# First of all, Welcome to Microverse :bouquet: :bouquet:

# Changes Requested :recycle:

almost there

## Hey @David-Kasilia :wave: 

### Highlights :1st_place_medal: 

Your project is working :clap:
Good commit history :heavy_check_mark:
Descriptive PR title :heavy_check_mark:
Following Github Flow :heavy_check_mark:
Following HTML, CSS best practices :heavy_check_mark:
Using .gitignore file :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

**You have made a great effort on your project :blush: and I can see that but you need to work on some issues to go to the next project but you are almost there. :+1:**

## 1
### Your Summary
third hello microverse project summary



### Descriptive Summary Example :heavy_check_mark:
clinton summary

- [x] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

## 2.
- [x] Your branch name is not descriptive enough. You are using **add-code-feature**, you can use **feature/initialize-project** or **feature/setup-project** :+1:

### Once you implement all the changes you should be ready to submit your project for a re-review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: stylesheet/style.css (hunk 2 above)
Comment (David-Kasilia) [reply stylesheet/style.css:6]: @HAMMAS-SALEEM the last error on linters . I have tried every possible way to fix it but still gives the same error. Though I have fixed the other pending issues.
Here is the error;
Unexpected missing end-of-source newline   no-missing-end-of-source-newline

**Review by topeogunleye — COMMENTED** [review]
Hi @David-Kasilia, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Merged and Deleted Pull Request ⚠️

- Pull request is merged and deleted. Please do not submit a merged and deleted pull request for review. Kindly revert your merging and deletion or create a new repo for your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Pull request closed without merging**